alembic/env.py

```diff
@@ -0,0 +1,83 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from database import Base
+
+from city.models import DBCity
+from temperature.models import DBTemperature
```

Comment:
`target_metadata = Base.metadata` serves as the entry point for your models, meaning you only need to import the `Base` object, not the individual models from different files. This works because all your models inherit from the same `Base` class, making them accessible through its metadata.